p4-16/psa/examples/psa-example-incremental-checksum.p4

```diff
@@ -82,6 +85,8 @@ parser IngressParserImpl(packet_in buffer,
                          out headers parsed_hdr,
                          inout metadata user_meta,
                          in psa_ingress_parser_input_metadata_t istd,
+                         in empty_metadata_t resubmit_meta,
```

Comment:
we really need optional/named arguments in the language spec. We shouldn't need to write this much boilerplate.

Reply:
We don't need to write it.  We just need to copy and paste it :-)